src/index.js

```diff
@@ -70,6 +68,8 @@ require('./scripts/store/dimSimpleItem.directive');
 require('./scripts/store/dimStats.directive');
 require('./scripts/store/dimFarming.directive');
 require('./scripts/store/dimClearNewItems.directive');
+require('./scripts/item-review/item-review.component');
+require('./scripts/move-popup/dimItemStats.component');
```

Comment:
i wanted to keep item-review separate because we may want to show reviews somewhere other than in just the move-popup